dataplane/src/main.rs

```diff
@@ -1,6 +1,120 @@
 // SPDX-License-Identifier: Apache-2.0
-// Copyright Open Network Fabric Authors
+
+use dpdk::dev::{Dev, TxOffloadConfig};
+use dpdk::eal::Eal;
+use dpdk::lcore::{LCoreId, WorkerThread};
+use dpdk::mem::{Pool, PoolConfig, PoolParams, RteAllocator};
+use dpdk::queue::rx::{RxQueueConfig, RxQueueIndex};
+use dpdk::queue::tx::{TxQueueConfig, TxQueueIndex};
+use dpdk::{dev, eal, socket};
+use net::packet::Packet;
+use net::parse::Parse;
+use tracing::{info, warn};
+
+#[global_allocator]
+static GLOBAL_ALLOCATOR: RteAllocator = RteAllocator::new_uninitialized();
+
+fn init(args: impl IntoIterator<Item = impl AsRef<str>>) -> Eal {
+    let rte = eal::init(args);
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::DEBUG)
+        .with_target(true)
+        .with_thread_ids(true)
+        .with_line_number(true)
+        .with_thread_names(true)
+        .init();
+    rte
+}
 
 fn main() {
-    // placeholder
+    let eal: Eal = init([
+        "--main-lcore",
+        "2",
+        "--lcores",
+        "2-4",
+        "--in-memory",
+        "--allow",
+        "0000:01:00.0,dv_flow_en=1",
+        "--huge-worker-stack=8192",
+        "--socket-mem=8192,0,0,0",
+        "--no-telemetry",
+        "--iova-mode=va",
```

Comment:
We plan to select some of these through some configuration mechanism later, right?